eth/_utils/merkle.py

```diff
@@ -64,16 +71,20 @@ def get_merkle_proof(tree: MerkleTree, item_index: int) -> Sequence[Hash32]:
 
 
 def _calc_parent_hash(left_node: Hash32, right_node: Hash32) -> Hash32:
-    """Calculate the parent hash of a node and its sibling."""
-    return keccak(left_node + right_node)
+    """
+    Calculate the parent hash of a node and its sibling.
+    """
+    return hash_eth2(left_node + right_node)
```

Comment:
Are these APIs dedicated to eth2.0?

Reply:
Currently, yes. It was written for proof of custody: https://github.com/ethereum/py-evm/pull/919